.github/workflows/pre-compile_llvm.yml

```diff
@@ -50,14 +50,17 @@ jobs:
           make --version
           ${{ matrix.cc }}-${{ matrix.version }} --version
           ${{ matrix.cpp }}-${{ matrix.version }} --version
-      
+      - name: Extract branch name
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
```

Comment:
Just curious, [GitHub documentation](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idoutputs) seems to suggest using the `::set-output name=branch::` syntax in this situation. Can you point me to the documentation that discusses the `##[set-output name=branch;]` syntax?

Also, the final `$(echo ${GITHUB_REF#res/heads/})` seems redundant. Couldn't the shell interpolate `${GITHUB_REF#refs/heads/}` within the first `echo` directly?

Reply:
Good point, @bryanpkc . The command works with one `echo` less and I changed to the yaml syntax (`::`) as suggested in the docs. 
I found the other approach on [stackoverflow](https://stackoverflow.com/questions/58033366/how-to-get-current-branch-within-github-actions), but you're right that it is not documented anywhere, so perhaps it is some outdated, but still working API. Github Actions are changing quite fast...
Thanks for the feedback!